packages/storybook/src/migrations/update-12-1-0/fix-storybook-tsconfig.ts

```diff
@@ -0,0 +1,97 @@
+import * as path from 'path';
+import {
+  chain,
+  Tree,
+  SchematicContext,
+  Rule,
+} from '@angular-devkit/schematics';
+
+import {
+  formatFiles,
+  readJsonInTree,
+  updateWorkspaceInTree,
+  serializeJson,
+} from '@nrwl/workspace';
+
+import { isFramework, TsConfig } from '../../utils/utilities';
+import { normalize } from '@angular-devkit/core';
+
+interface ProjectDefinition {
+  root: string;
+  sourceRoot: string;
+  projectType: 'library' | 'application';
+
+  schematic?: Record<string, any>;
+  architect: Record<
+    string,
+    import('@angular-devkit/core').workspaces.TargetDefinition
+  >;
+}
+
+export default function (tree: Tree, context: SchematicContext) {
+  return chain([update, formatFiles()]);
+}
+
+function update(tree: Tree, context: SchematicContext): Rule {
+  return updateWorkspaceInTree((config, context, tree) => {
```

Comment:
I think this will look much neater using `@nrwl/devkit`.

The following gets the projects from the workspace:

```ts
import { getProjects } from '@nrwl/devkit';
```

Reply:
I can update if you want...just did basically copy & paste programming for the quick migration 😉